server/cluster/cluster.go

```diff
@@ -1,56 +1,72 @@
 package cluster
 
 import (
+	"io/ioutil"
+
 	"github.com/smancke/guble/protocol"
+	"github.com/smancke/guble/server/store"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/hashicorp/memberlist"
 
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"net"
 	"strconv"
 )
 
+var (
+	ErrNodeNotFound = errors.New("Node not found.")
+)
+
 // Config is a struct used by the local node when creating and running the guble cluster
 type Config struct {
-	ID                   int
+	ID                   uint8
 	Host                 string
 	Port                 int
 	Remotes              []*net.TCPAddr
 	HealthScoreThreshold int
 }
 
-// MessageHandler is an interface used for handling messages in cluster.
+// router interface specify only the methods we require in cluster from the Router
+// router is an interface used for handling messages in cluster.
 // It is logically connected to the router.Router interface, by reusing the same func signature.
```

Comment:
this comment-line can be removed now, the other 2 lines offer a better explanation